src/lib/controllers/UsersController.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import type { Repository, User, UserCredentials } from '../../types';
+
+import Controller from './Controller';
+import httpVerb from '../decorators/httpVerb';
+import route from '../decorators/route';
+
+class UsersController extends Controller {
+  _usersRepository: Repository<User>;
+
+  constructor(usersRepository: Repository<User>) {
+    super();
+    this._usersRepository = usersRepository;
+  }
+
+  @httpVerb('post')
+  @route('/v1/users')
```

Comment:
decorators don't work for `method = () =>` arrow functions, so we have to use this syntax?